package.json

```diff
@@ -74,6 +74,7 @@
       "**/*.(t|j)s"
     ],
     "coverageDirectory": "../coverage",
+    "coveragePathIgnorePatterns": ["index.ts"],
```

Comment:
No related to this PR, but it was easy and quick. We also have other file patterns which could be ignored by the coverage report (`main.ts`, `app.module.ts`), what do you think on flagging those?